torch/_dynamo/optimizations/backends.py

```diff
@@ -788,14 +767,35 @@ def fwd(*args):
     return fwd
 
 
-def ipex_fp32(gm: torch.fx.GraphModule, example_inputs):
-    kwargs_ipex = {"datatype": "fp32"}
-    return BACKENDS["ipex"](gm, example_inputs, **kwargs_ipex)
+@create_backend
+def ipex(subgraph):
+    import intel_extension_for_pytorch  # type: ignore[import]  # noqa: F401
```

Comment:
Do "try-catch" and fall back to eager on exception?

Reply:
I checked other backends like onnx backend, they also don't have fall back path. So I'm not sure if it needs to be added.

Reply:
I saw the "tvm" backend does fallbacks. Anyway, at least, we'd better print a readable error message on import error?

Reply:
have added a readable error message, thanks!